### PR TITLE
Update rspec readme

### PR DESCRIPTION
**Pull request by ChrisBr**

### CODE REVIEW

File: src/api/app/views/webui2/webui/project/show.html.haml [hunk 1]
```diff
@@ -0,0 +1 @@
+Hello World
```
Comment (DavidKang) [inline src/api/app/views/webui2/webui/project/show.html.haml]: This file is not related to this PR. Could you remove it?

Comment (ChrisBr) [reply src/api/app/views/webui2/webui/project/show.html.haml]: Argh! Removed

**Comment by ChrisBr** [comment]
Added do not merge label as I misuse this PR to test the review apps :smile:

File: src/api/spec/README.md [hunk 2]
```diff
@@ -144,18 +144,16 @@ Attention: Faker generates random but **NOT** unique data!
...
-```
-docker-compose run --rm frontend bundle exec rake db:fixtures:load RAILS_ENV=test
-docker-compose run --rm frontend script/start_test_backend
-```
-
 We use [VCR](https://github.com/vcr/vcr) to record the response from the backend.
 VCR records the HTTP interactions with the backend and replays them during future test runs for fast, deterministic, accurate tests.
 Once your test ran successfully for the first time [VCR](https://github.com/vcr/vcr) will have recorded a new cassette (a simple yml file) in `spec/cassettes`.
 
+You can log into the frontend container which already has a backend running with:
```
Comment (hennevogel) [inline src/api/spec/README.md]: And why would I?

File: src/api/spec/README.md [hunk 3]
```diff
@@ -187,14 +185,11 @@ You should only use shared examples where you have the exact same functionality
 Otherwise these tests get fast hard to refactor and review.
 In our experience, shared examples are used mainly for controllers. Since models are pretty different from each other, they (usually) do not share much logic.
 
-### Travis
-We use [travis-ci](https://travis-ci.org/) for continues integration.
+### Continues Integration
```
Comment (hennevogel) [inline src/api/spec/README.md]: ~~Continues~~ -> Continuous

File: src/api/spec/README.md (hunk 2 above)
Comment (ChrisBr) [reply src/api/spec/README.md]: To run rspec inside the container ;) You mean I should add how I can run rspec inside the container?

Comment (hennevogel) [reply src/api/spec/README.md]: Well it's not clear what you are trying to say. And in general I would leave docker stuff out of this...

Comment (coolo) [reply src/api/spec/README.md]: what is your recommended way to update vcrs? The one @ChrisBr outlined, worked for me

Comment (hennevogel) [reply src/api/spec/README.md]: I'm not saying to use any other way. I'm saying that this document shouldn't include any docker specifics (only normal commands) and should refer to the wiki for how to run them in the development environment. 

That way we have one place to keep up to date.

File: src/api/spec/README.md (hunk 3 above)
Comment (dmarcoux) [reply src/api/spec/README.md]: There's the same typo one line below too

File: src/api/spec/README.md (hunk 2 above)
Comment (ChrisBr) [reply src/api/spec/README.md]: Ok! Just added it because before it was also already there! I will remove it and move it to our development setup wiki.

File: src/api/spec/README.md (hunk 3 above)
Comment (ChrisBr) [reply src/api/spec/README.md]: Argh right

**Comment by ChrisBr** [comment]
@hennevogel updated

**Comment by ChrisBr** [comment]
I also splitted the development environment page and updated it:

https://github.com/openSUSE/open-build-service/wiki/Development-Environment-Overview
https://github.com/openSUSE/open-build-service/wiki/Development-Environment-Tips-&-Tricks

cc @hennevogel @coolo

File: src/api/spec/README.md [hunk 4]
```diff
@@ -187,14 +179,13 @@ You should only use shared examples where you have the exact same functionality
...
+### Continuous Integration
+We use [travis-ci](https://travis-ci.org/) and [CircleCI](https://circleci.com) for continuous Integration.
 
 #### Setup
-As travis-ci runs on an Ubuntu machine, we need to add the OBS repository and install some OBS specific Ubuntu packages first.
-We do this in [dist/ci/obs_testsuite_travis_install.sh](https://github.com/openSUSE/open-build-service/blob/master/dist/ci/obs_testsuite_travis_install.sh).
-You can find the Ubuntu specific packages in this repository [http://download.opensuse.org/repositories/OBS:/Server:/Unstable/xUbuntu_12.04/](http://download.opensuse.org/repositories/OBS:/Server:/Unstable/xUbuntu_12.04/).
-We do not package the rubygems for Ubuntu, instead we use bundler to install them.
+In both services we use docker container for running our tests. The docker container are built with OBS in the container subprojects of [O:S:U](https://build.opensuse.org/project/subprojects/OBS:Server:Unstable) (e.g. https://build.opensuse.org/project/show/OBS:Server:Unstable:container:SLE12:SP3). With this approach we can test on our supported platforms like openSUSE or SLE and easily migrate to new platforms if necessary.
```
Comment (eduardoj) [inline src/api/spec/README.md]: I would add a `s` at the end of `we use docker container` and `The docker container`.

File: src/api/spec/README.md [hunk 5]
```diff
@@ -187,14 +179,13 @@ You should only use shared examples where you have the exact same functionality
 Otherwise these tests get fast hard to refactor and review.
 In our experience, shared examples are used mainly for controllers. Since models are pretty different from each other, they (usually) do not share much logic.
 
-### Travis
-We use [travis-ci](https://travis-ci.org/) for continues integration.
+### Continuous Integration
+We use [travis-ci](https://travis-ci.org/) and [CircleCI](https://circleci.com) for continuous Integration.
```
Comment (dmarcoux) [inline src/api/spec/README.md]: Tiny change, `Integration` should not be capitalized at all.
